src/nih_wayfinding/app/scripts/routing/directions-service.js

```diff
@@ -35,6 +37,38 @@
                 return RoutingResponseStub;
             }, 100, false);
         }
+
+        function getFlagIconName(flagType) {
+            // TODO: Write tests once actual icons exist
+            switch (flagType) {
+                case 'bench':
+                    return 'glyphicon-flash';
```

Comment:
Careful, last time I used this icon as a placeholder, it never ended up getting replaced. Electrified benches :)